packages/browser/src/ClientAuthentication.ts

```diff
@@ -34,8 +34,9 @@ import {
   ISessionInfo,
   ISessionInfoManager,
 } from "@inrupt/solid-client-authn-core";
-import URL from "url-parse";
+import UrlParse from "url-parse";
```

Comment:
I'm not sure about this, but it seems to me that the only export from the `url-parse` library is actually named `URLParse` (note the capitalization), so changing that capitalization seems a little disingenuous(!): 
```suggestion
import URLParse from "url-parse";
```

Reply:
We are using the default import from `url-parse`, so the name we give it is up to us, and in its current form, it is aligned with our naming convention.

Reply:
Hhhmmm.... I can't say I agree with that argument (but then again, maybe this genuinely is just idiomatic JavaScript common practice - but if so, I'd be surprised (and disappointed!)).
Right now, we're simply using this 3rd-party library to perform functionality that we need, specifically to be able to parse an IRI into it's constituent parts. Sure, we can rename this class to whatever we like, but why would we want to rename a 3rd-party class (and such a popularly used one at that)? Isn't that just going to introduce potential confusion (i.e. it certainly confused me already!)...?
Given it's a popular 3rd-party library (actually, regardless of it's popularity, but being popular emphasizes the point I suppose), I see no reason to rename it just to conform to our particular naming conventions - in fact I'd see that as a distinctly bad idea ('cos doing so makes it appear to new developers as one of 'our classes', when in fact it's not).
I don't know if this helps, but it strikes me as an argument for arbitrarily renaming something super-popular like Moment as `import DateUtils from "moment";`, just 'cos I have a date-centric naming convention in my particular codebase.
I'd be interested to hear @Vinnl 's or @ajacksified 's thoughts on this...?